addons/website/views/snippets/snippets.xml

```diff
@@ -87,6 +87,9 @@
                 <t t-snippet="website.s_three_columns" string="Columns" group="columns">
                     <keywords>columns, description</keywords>
                 </t>
+                <t t-snippet="website.s_wavy_grid" string="Wavy Grid" group="columns">
```

Comment:
As discussed, don't forget to rebase and move the declaration of your snippet [to the right place](https://github.com/odoo/odoo/blob/master/addons/website/views/snippets/snippets.xml#L65)